app/components/Poll/index.tsx

```diff
@@ -51,197 +51,243 @@ type OptionEntityRatio = OptionEntity & {
           </Flex>
         </div>
       </Flex>
-    </Card>
+      <Flex
+        alignItems="center"
+        justifyContent="center"
+        className={styles.bottomBar}
+        style={{ cursor: alwaysOpen ? '' : 'pointer' }}
+        onClick={alwaysOpen ? undefined : toggleExpanded}
+      >
+        {!alwaysOpen && (
+          <Icon
+            className={styles.arrowIcon}
+            size={26}
+            name={expanded ? 'chevron-up' : 'chevron-down'}
+          />
+        )}
+      </Flex>
+    </Flex>
   );
 };
 
 export default Poll;
+
+type VoteOpenProps = {
+  poll: PollEntity;
+  details?: boolean;
+  options: OptionEntityRatio[];
+};
+
+const VoteOpen = ({ details, poll, options }: VoteOpenProps) => {
+  const dispatch = useAppDispatch();
+
+  return (
+    <Flex column alignItems="center" className={styles.optionWrapper}>
+      {details && <p className={styles.description}>{poll.description}</p>}
+      {options.map((option) => (
+        <Button
+          key={option.id}
+          className={styles.voteButton}
+          dark
+          onClick={() => dispatch(votePoll(poll.id, option.id))}
+        >
+          {option.name}
+        </Button>
+      ))}
+    </Flex>
+  );
+};
+
+type VoteResultsProps = {
+  poll: PollEntity;
+  details?: boolean;
+  options: OptionEntityRatio[];
+  resultsHidden: boolean;
+};
+
+const VoteResults = ({
+  details,
+  poll,
+  options,
+  resultsHidden,
+}: VoteResultsProps) => (
+  <Flex column className={styles.optionWrapper}>
+    {details && <p className={styles.description}>{poll.description}</p>}
+    <table className={styles.pollTable}>
+      <tbody>
+        {options.map(({ id, name, votes, ratio }) => {
+          return (
+            <tr key={id}>
+              <td className={styles.textColumn}>{name}</td>
+              <td className={styles.graphColumn}>
+                {votes === 0 ? (
+                  <span className="secondaryFontColor">Ingen stemmer</span>
+                ) : (
+                  <div className={styles.fullGraph}>
+                    <div
+                      style={{
+                        width: `${ratio}%`,
+                      }}
+                    >
+                      <div className={styles.pollGraph}>
+                        {ratio >= 18 && <span>{`${ratio}%`}</span>}
+                      </div>
+                    </div>
+                    {ratio < 18 && (
+                      <span
+                        style={{
+                          padding: '5px',
+                          marginLeft: '2px',
+                        }}
+                      >
+                        {`${ratio}%`}
+                      </span>
+                    )}
+                  </div>
+                )}
+              </td>
+            </tr>
+          );
+        })}
+      </tbody>
+    </table>
+    {resultsHidden && (
+      <div className={styles.resultsHiddenInfo}>
+        Resultatet er skjult for vanlige brukere.
```

Comment:
I see you've added a period here - intentional?

Reply:
Nope:) a lot of this is stuff that I haven't looked thoroughly through - as it took enough time to fit in the changes from the original PR to get it to work(: so your comments are very much appreciated!